src/dscanner/dscanner_version.d

```diff
@@ -10,10 +10,11 @@ module dscanner.dscanner_version;
  */
 enum DSCANNER_VERSION = "v0.4.0";
 
-version (Windows)
+version (built_with_dub)
 {
+	enum GIT_HASH = import("dubhash.txt");
```

Comment:
FYI: Left it like this for now - it will conflict with https://github.com/dlang-community/D-Scanner/pull/605 anyhow.